nautobot_circuit_maintenance/__init__.py

```diff
@@ -82,5 +82,13 @@ def ready(self):
         post_migrate.connect(custom_fields_extension, sender=self)
         post_migrate.connect(import_notification_sources, sender=self)
 
+        # pylint: disable=import-outside-toplevel
+        from nautobot_capacity_metrics import register_metric_func
+        from .metrics_app import metric_circuit_operational
```

Comment:
Possibly move these inside the below `if` block? No need to import them if they're not going to be used.

Reply:
exactly